dist/style.css

```diff
@@ -59,3 +55,288 @@ body {
   -ms-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+/* container css for most high level containers  */
+.container-parent {
+  height: 100%;
+  position: relative;
+}
+
+.container-header {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+}
+
+.container-navSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding-bottom: 4%;
+}
+
+.container-nav {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 5%;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.container-moto {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 100%;
+  justify-content: left;
+}
+
+/* effects for hover on buttons */
+.menu {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  background-color: white;
+  text-align: center;
+  padding: 2%;
+  border-radius: 25px;
+  border: 2px solid #A9A9A9;
+  width: 25%;
+}
+
+.menu:hover {
+  background-color: #662A61;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+}
+
+.fas:hover {
+  background-color: #562554;
+  color: white;
+}
+
+p {
+  margin: 0px;
+  margin-left: 7.5%;
+  margin-right: 10.5%;
+}
+
+fas {
+  height: 0%;
+}
+
+/* logo resizing  */
+.container-logo {
+  max-width: 65%;
+  height: auto;
+}
+
+#mainlogo {
+  max-width: 65%;
+  height: auto;
+  padding-right: -25%;
+}
+
+
+/* main content in split boxes  */
+.container-body {
+  margin-bottom: 20px;
+}
+
+.content {
+  font-family: 'AlexandriaFLF';
+  display: flex;
+  flex-direction: row;
+  border: 5px solid #562554;
+  border-radius: 25px;
+  background-color: white;
+}
+
+.content-left {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+}
+
+.content-right {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+  width: 50%;
+  align-items: center;
+  text-align: center;
+}
+
+/* form styling  */
+input {
```

Comment:
Maybe we can update the font here as well? I propose that we do it here rather than in the `::placeholder` selector so that it affects both the placeholder text and the actual user input.

Reply:
Also, in the mockup, it looks like the input field text should have a little `margin` or `padding` on its left.

Reply:
added padding and changed font within the input field - changed font on send button as well